.github/bicep/main.bicep

```diff
@@ -44,12 +48,17 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2022-05-01' = {
   }
 }
 
+resource blobService 'Microsoft.Storage/storageAccounts/blobServices@2022-05-01' = {
+  parent: storageAccount
+  name: 'default'
+}
+
 resource hostingPlan 'Microsoft.Web/serverfarms@2021-03-01' existing = {
   name: hostingPlanName
   scope: resourceGroup(hostingPlanRgName)
 }
 
-resource kv 'Microsoft.KeyVault/vaults@2023-07-01' = {
```

Comment:
why was this KV version changed to use an older version? @calumjs

Reply:
![image](https://github.com/user-attachments/assets/88c3d4f0-6aa8-4743-9364-eb6c666b36cd)
When I tried to build it locally during testing I got a bunch of warnings, so I just swapped it to a value from another pipeline.
Did I miss something?

Reply:
Ah ok I see - I just had an old version of bicep installed - no warning after upgrade.
Will put it back to the 2023 version